dev/com.ibm.ws.security.authorization.jacc.ejb/src/com/ibm/ws/security/authorization/jacc/ejb/impl/EJBJaccServiceImpl.java

```diff
@@ -204,13 +205,13 @@ public boolean isSubjectInRole(String applicationName, String moduleName, String
 
     private boolean checkResourceConstraints(String contextId, List<Object> methodParameters, EnterpriseBean bean, Permission ejbPerm, Subject subject, PolicyProxy policyProxy) {
         boolean result = false;
-        final Object[] ma = null;
+        Object[] ma = null;
 
         /*
          * TODO Doesn't seem to handle EJB-3.0 annotated beans.
          */
         if (methodParameters != null && methodParameters.size() > 0) {
-            methodParameters.toArray(new Object[methodParameters.size()]);
+            ma = methodParameters.toArray(new Object[methodParameters.size()]);
```

Comment:
This is the follow on change from https://github.com/OpenLiberty/open-liberty/pull/30293#discussion_r1907899673.  Do you believe this should be covered in a `release bug` issue?

Reply:
Yes, I think this merits a release bug. This will fix the behavior, so it would be good to have it documented in the release notes.

Reply:
Issue #30890 opened